praw/models/reddit/subreddit.py

```diff
@@ -304,6 +420,29 @@ def unread(self, **generator_kwargs):
             self.subreddit._reddit, API_PATH['moderator_unread'].format(
                 subreddit=self.subreddit), **generator_kwargs)
 
+    def update_settings(self, **kwargs):
```

Comment:
Can we just call this `update`? That should be consistent with the other models in PRAW4.